From 4f8dc63917f67615655bdada31d1e9f4cf7a1412 Mon Sep 17 00:00:00 2001
From: Arie Brown <abrown@habana.ai>
Date: Sun, 09 Jan 2022 20:56:15 +0200
Subject: [PATCH] Revert "[SW-69272] pci: update msix pba value"

although the content of this patch is correct, it is being reverted
to align with master.

This reverts commit da8fc2ff6f2e9782b8d1a7b9c8ba6fe5f1242781.

Change-Id: I4cc2b0720d06254b9667600d77a44c14fc3b18c1
Signed-off-by: Arie Brown <abrown@habana.ai>
---

diff --git a/arch/gaudi2/common/common.c b/arch/gaudi2/common/common.c
index e0dddaf..62db4c9 100644
--- a/arch/gaudi2/common/common.c
+++ b/arch/gaudi2/common/common.c
@@ -709,7 +709,7 @@
 	}
 }
 
-#define MSIX_PBA_OFFSET_REG_VAL		0x2002
+#define MSIX_PBA_OFFSET_REG_VAL		0x400
 
 void set_pci_msix_pba_offset_val(void)
 {
